src/components/application_manager/test/message_helper/message_helper_test.cc

```diff
@@ -37,23 +37,41 @@
 #include "utils/macro.h"
 #include "utils/make_shared.h"
 #include "application_manager/policies/policy_handler.h"
-#include "application_manager/test/resumption/include/application_mock.h"
+#include "application_manager/test/include/application_manager/application_mock.h"
 #include "utils/custom_string.h"
-#include "policy/mock_policy_settings.h"
 #include "application_manager/policies/policy_handler.h"
-#include "application_manager/mock_application_manager.h"
+#include "application_manager/test/include/application_manager/mock_application.h"
```

Comment:
@indlu As for me - it looks strange that we have `application_manager/test/include/application_manager/application_mock.h` and `application_manager/test/include/application_manager/mock_application.h`